src/error.ts

```diff
@@ -26,6 +27,11 @@ export default function inspectObject(error: Error, options: Options) {
   }
   message = message ? `: ${message}` : ''
   options.truncate -= message.length + 5
+  options.seen = options.seen || []
+  if (options.seen.indexOf(error) >= 0) {
```

Comment:
What about:

```suggestion
  if (options.seen.includes(error)) {
```

I find it a bit more readable.

Reply:
i just copied what we have in the object inspection code

I'm happy to change it in both places but wasn't sure if there's some compat reason we're not using `includes` (probably just old code?)

Reply:
its been available since node 6 so I'm gonna go ahead with it 😅